src/tactic/command/global_search_trigger.py

```diff
@@ -111,13 +111,12 @@ def execute(my):
             
             update_data = input.get("update_data")
 
-            # If Relative dir is changed, update path keywords
-            if "relative_dir" in update_data and not input.get("mode") == "insert":
-                new_path = update_data.get("relative_dir")
+            # If Relative dir is changed or file is renamed, update path keywords
+            if "relative_dir" in update_data or "name" in update_data and not input.get("mode") == "insert":
+                file_path = input.get("sobject").get("relative_dir")
                 asset_name = input.get("sobject").get("name")
-                if asset_name:
-                    new_path = "%s/%s" % (new_path, asset_name)
```

Comment:
Celton just added this change to ensure that the asset_name was included in the path so that it would use it as a keyword. Please confirm this with Celton. Does the file_path already include the asset name?